### ISSUE TITLE
Error: Cannot find module '@hugoalh/advanced-determine'

### ISSUE THREAD

**Issue by BambusRatteTV** [hugoalh/send-discord-webhook-ghaction#116]
Hey there, in Version 3.1.1 Run act will not end as planned, it drops the current failure:

```
[0-discord-test-rss.yml/Send To Discord 0]   🐳  docker exec cmd=[node /var/run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1/main.js] user=
[0-discord-test-rss.yml/Send To Discord 0]   | internal/modules/cjs/loader.js:888
[0-discord-test-rss.yml/Send To Discord 0]   |   throw err;
[0-discord-test-rss.yml/Send To Discord 0]   |   ^
[0-discord-test-rss.yml/Send To Discord 0]   | 
[0-discord-test-rss.yml/Send To Discord 0]   | Error: Cannot find module '@hugoalh/advanced-determine'
[0-discord-test-rss.yml/Send To Discord 0]   | Require stack:
[0-discord-test-rss.yml/Send To Discord 0]   | - /run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1/main.js
[0-discord-test-rss.yml/Send To Discord 0]   |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Function.Module._load (internal/modules/cjs/loader.js:730:27)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Module.require (internal/modules/cjs/loader.js:957:19)
[0-discord-test-rss.yml/Send To Discord 0]   |     at require (internal/modules/cjs/helpers.js:88:18)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Object.<anonymous> (/run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1/main.js:6:27)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Module._compile (internal/modules/cjs/loader.js:1068:30)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Module.load (internal/modules/cjs/loader.js:933:32)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Function.Module._load (internal/modules/cjs/loader.js:774:14)
[0-discord-test-rss.yml/Send To Discord 0]   |     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
[0-discord-test-rss.yml/Send To Discord 0]   |   code: 'MODULE_NOT_FOUND',
[0-discord-test-rss.yml/Send To Discord 0]   |   requireStack: [
[0-discord-test-rss.yml/Send To Discord 0]   |     '/run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1/main.js'
[0-discord-test-rss.yml/Send To Discord 0]   |   ]
[0-discord-test-rss.yml/Send To Discord 0]   | }
Error: exit with `FAILURE`: 1
[0-discord-test-rss.yml/Send To Discord 0]   ❌  Failure - hugoalh/GitHubAction.SendToDiscord@v3.1.1
```

**Comment by hugoalh** [comment]
Unfortunately this action is not officially supported Docker environment (as mentioned in `./README.md`, only NodeJS v12.13.0 or above) (but pending to support Docker in the future). Of course [Act](https://github.com/nektos/act) is also not officially supported.

Here is some possible solutions to resolve this issue: (all solution required network access)

### Solution 1

1. Change directory to action location (in your case it is `/var/run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1`).
2. Execute `npm ci` or `npm install`.
3. Do `docker exec cmd=[node /var/run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1/main.js]` again.

### Solution 2

1. Do `docker exec cmd=[node /var/run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1/setup.js]`.
2. Do `docker exec cmd=[node /var/run/act/actions/hugoalh-GitHubAction.SendToDiscord@v3.1.1/main.js]` again.

**Comment by hugoalh** [comment]
This issue is now closed due to v4.0.0 support Docker as entrypoint; If this issue happened again, please reopen this issue.
